src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,100 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.exception.UserAgeException;
+import core.basesyntax.exception.UserLoginExistsException;
+import core.basesyntax.exception.UserLoginLengthException;
+import core.basesyntax.exception.UserPasswordLengthException;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private final RegistrationService registrationService = new RegistrationServiceImpl();
+
+    @Test
+    @DisplayName("User register with null login")
+    void register_nullLogin_throwNullPointerException() {
+        User userWithNullLogin = new User(null, "1234567", 45);
+        assertThrows(NullPointerException.class,
+                     () -> registrationService.register(userWithNullLogin));
+    }
+
+    @Test
+    @DisplayName("User register with null password")
+    void register_nullPassword_throwNullPointerException() {
+        User userWithNullPassword = new User("ivanivanov12@gmail.com", null, 45);
+        assertThrows(NullPointerException.class,
+                     () -> registrationService.register(userWithNullPassword));
+    }
+
+    @Test
+    @DisplayName("User register with null age")
+    void register_nullAge_throwNullPointerException() {
+        User userWithNullAge = new User("ivanivanov12@gmail.com", "1234567", null);
+        assertThrows(NullPointerException.class,
+                     () -> registrationService.register(userWithNullAge));
+    }
+
+    @Test
+    @DisplayName("User register correct user case #1")
+    void register_correctUser_ok() {
+        User user = new User("mykhailo777@gmail.com", "1234567", 23);
+        User actualUser = registrationService.register(user);
+        User expectedUser = new User("mykhailo777@gmail.com", "1234567", 23);
+        assertEquals(actualUser, expectedUser);
```

Comment:
This test does not verify if the user was actually added to the storage. You should check the storage to ensure the user is present after registration.